script.js

```diff
@@ -1,4 +1,180 @@
+var numberofDice = 2;
+var player1Array = [];
+var player2Array = [];
+var roundCounter = 0;
+var player1RoundCounter = 0;
+var player2RoundCounter = 0;
+var player1;
+var player1CombinedNumber = 0;
+var player2CombinedNumber = 0;
+var myOutputValue = "";
+var player1TotalScore = 0;
+var player2TotalScore = 0;
+
+// generate 1 dice roll (1<x<6) - T
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  return randomNumber;
+};
+
+// generate the dice rolls for each player and create an array - T
+var createDiceRollArray = function (numberofDice) {
+  var diceRollArray = [];
+  var diceRollCounter = 0;
+  while (diceRollCounter < numberofDice) {
+    diceRollArray.push(diceRoll());
+    diceRollCounter += 1;
+  }
+  return diceRollArray;
+};
+
+var concatenate2Numbers = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //when player 1 clicks submit, display the results of dice roll
+  //if player 1 and player 2 have both gone, then start from player 1 again
```

Comment:
Good use of comments. In this case, the comment was really crucial to helping the reader understand how you interpret the modes.